ui/raidboss/data/02-arr/raid/t10.js

```diff
@@ -106,6 +106,8 @@
         'Spike Flail': 'Dornendresche',
         'Wild Charge': 'Wilde Rage',
       },
+      '~effectNames': {
+      },
```

Comment:
Erhöhte Verwundbarkeit
Erratischer Puls
Markiert
Stromschlag

Reply:
again, they are not used in either txt/js file so no need to include them.